src/solver/modeler/api/include/antares/solver/modeler/api/linearProblemData.h

```diff
@@ -28,20 +28,33 @@
 namespace Antares::Solver::Modeler::Api
 {
 
-class LinearProblemData
+class ILinearProblemData
```

Comment:
Interface class **ILinearProblemData** was created here, but it should be eventually moved in a separate header file.
The current location (**solver/modeler/api**) seems the right place for this header file, because this interface belongs to and should be defined where the contract it represents is actually required : class **LinearProblemBuilder** and class **LinearProblemFiller**